sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithValueStateTTLSuite.scala

```diff
@@ -160,6 +163,30 @@ case class MultipleValueStatesTTLProcessor(
   }
 }
 
+// Class to verify state schema is correctly written for state vars.
+class TTLProcessorWithCompositeTypes(
+      ttlKey: String,
+      noTtlKey: String,
+      ttlConfig: TTLConfig)
+  extends MultipleValueStatesTTLProcessor(
+    ttlKey: String, noTtlKey: String, ttlConfig: TTLConfig) {
```

Comment:
nit: any specific reason to put type?

Reply:
Ah no specific reason, just removed them.